### PR TITLE
Updated household file with MU of labor

### PR DESCRIPTION
**Pull request by Devashish-Thakar**
Hi All,

Please review the updated file with MU of labor. If required, I will update it with your suggestions.

Thanks,
Devashish

### CODE REVIEW

**Comment by LeiliPR** [comment]
> Hi All,
> 
> Please review the updated file with MU of labor. If required, I will update it with your suggestions.
> 
> Thanks,
> Devashish

good work Devavish, Thanks.
I just looked over it briefly, have you considered exogeneous labor supply? am I right?

**Comment by jdebacker** [comment]
@devashish2019 You'll want to write your code in a `.py` file not a Jupyter Notebook as you are trying to commit here.

File: scripts/households_v1.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+def get_c(r, w, n, b_s, b_sp1):
+    '''
+    Use the budget constraint to solve for consumption
+    '''
+    c = w * n + (1 + r) * b_s - b_sp1
+
+    return c
+
+
+def u_prime(c, sigma):
```
Comment (jdebacker) [inline scripts/households_v1.py]: Since you have an `mu_labor` function now, maybe this should be `mu_cons` (or something like that) to be distinct?

**Comment by jdebacker** [comment]
@devashish2019  you can remove the notebook from this PR with:

`git rm --cached households_v1.ipynb`

File: scripts/households_v1.py (hunk 1 above)
Comment (Devashish-Thakar) [reply scripts/households_v1.py]: Thanks Jason. I thought I should not disturb the existing names, but I can do that and push the updated file

**Comment by Devashish-Thakar** [comment]
@jdebacker I have removed the jupyter notebook. It does not show up on my branch now

**Comment by jdebacker** [comment]
@devashish2019 Why the `_v1` in the file name?  There is no other script with the names `households.py` in this repo and we are using git as a version control software so no need to keep track of version names in the filename.

FYI, to rename the file on git:
`git move households_v1.py households.py`

**Comment by jdebacker** [comment]
This looks ready to go.  I'll merge it in.

**Comment by LeiliPR** [comment]
Hi,
I am trying to push these Two files, but nothing happens after I do push command!

Leili

**Comment by Devashish-Thakar** [comment]
@LeiliPR For whatever new files you are trying to add from your end, you need to 
--> create a branch on your local repo
-->Commit changes to the branch and then
-->Do a pull request

**Pull request merged**